### PR TITLE
fix: absolute path issue in link for consistent URLs

### LINKED ISSUE #3855: React Native CLI invalid documentation link
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

Open react native cli documentation opens "Page Not Found" because of invalid href:
 

### Expected behavior

To open react native CLI docs

### Describe the bug

Invalid href of "React Native CLI" card element

### Steps To Reproduce

```Markdown
1. Open https://reactnativeelements.com/docs/
2. Click on React Native CLI card
```

### Screenshots

image

### Your Environment

N/A

### PR DESCRIPTION
**Pull request by abdulbasithqb**
## Motivation

Fixes #3855 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Checked with `example` app

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: website/versioned_docs/version-4.0.0-rc.8/index.mdx
Comment (khushal87) [inline website/versioned_docs/version-4.0.0-rc.8/index.mdx:1]: You don't have to update the versioned docs. Please remove the change and this PR will be good to go. Thanks 😄

**Review by khushal87 — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-4.0.0-rc.8/index.mdx
Comment (abdulbasithqb) [reply website/versioned_docs/version-4.0.0-rc.8/index.mdx:1]: Thanks for the review, revered the changes

**Review by khushal87 — APPROVED** [review]

**Comment by khushal87** [comment]
Please update the branch with the base branch so that I can merge the PR.

**Comment by abdulbasithqb** [comment]
> Please update the branch with the base branch so that I can merge the PR.

Done

**Pull request merged**